src/gui/dialogs/preferences_dialog.cpp

```diff
@@ -360,7 +360,7 @@ void preferences_dialog::post_build(window& window)
 
 	/* ACCELERATED SPEED */
 	register_bool("turbo_toggle", true, turbo, set_turbo,
-		[&](widget& w) { disable_widget_on_toggle<slider>(window, w, "turbo_slider"); }, true);
+		[](widget&) { }, true);
```

Comment:
If you're disabling the callback, use the 4-argument version of `register_bool` and remove these last two argument completely.

Reply:
I don't see how I can use the other overload of `register_bool` since I don't have a `bool` variable to pass for its third argument, but I'll remove the last two arguments.

Reply:
Ah, yes. Sorry. You can't use the 4-argument version, but you should still drop the last two arguments.

Reply:
Done and pushed